goes13-l3-imager.json

```diff
@@ -0,0 +1,208 @@
+            "description": "Midlevel atmospheric water vapor, winds, rainfall",
+            "center_wavelength": 6.55,
+            "min_value": 19000,
+            "max_value": 27000,
+            "nodata": 32767,
+            "scale": 0.01,
+            "data_type": "int16",
+            "mime_type": "application/octet-stream",
+            "resolution_x": 0.04,
+            "resolution_y": 0.04,
+            "resolution_unit": "Degree",
+            "full_width_half_max": 1.3806
+        },
+        {
+            "name": "B04",
+            "description": "Surface and clouds",
+            "center_wavelength": 10.7,
+            "min_value": 19000,
+            "max_value": 32700,
+            "nodata": 32767,
+            "scale": 0.01,
+            "data_type": "int16",
+            "mime_type": "application/octet-stream",
+            "resolution_x": 0.04,
+            "resolution_y": 0.04,
+            "resolution_unit": "Degree",
+            "full_width_half_max": 0.9835
+        },
+        {
+            "name": "B05",
+            "description": "Air temperature, cloud heights",
+            "center_wavelength": 13.35,
+            "min_value": 19000,
+            "max_value": 32700,
+            "nodata": 32767,
+            "scale": 0.01,
+            "data_type": "int16",
+            "mime_type": "application/octet-stream",
+            "resolution_x": 0.04,
+            "resolution_y": 0.04,
+            "resolution_unit": "Degree",
+            "full_width_half_max": 0.7005
+        }
+    ],
+    "summaries": {
+        "platform": [
+            "goes-13"
+        ],
+        "constellation": [
+            "goes"
+        ],
+        "instruments": [
+            "imager"
+        ]
+    },
+    "item_assets": {
+        "B01": {
+            "type": "application/gzip",
+            "title": "GOES-13/Imager Band 01 (VIS)",
+            "roles": [
+                "data"
+            ]
+        },
+        "B02": {
+            "type": "application/gzip",
+            "title": "GOES-13/Imager Band 02 (IR)",
+            "roles": [
+                "data"
+            ]
+        },
+        "B03": {
+            "type": "application/gzip",
+            "title": "GOES-13/Imager Band 03 (IR)",
+            "roles": [
+                "data"
+            ]
+        },
+        "B04": {
+            "type": "application/gzip",
+            "title": "GOES-13/Imager Band 04 (IR)",
+            "roles": [
+                "data"
+            ]
+        },
+        "B05": {
+            "type": "application/gzip",
+            "title": "GOES-13/Imager Band 05 (IR)",
+            "roles": [
+                "data"
+            ]
+        },
+        "thumbnail": {
+            "type": "image/png",
+            "title": "Thumbnail",
+            "roles": [
+                "thumbnail"
+            ]
+        }
+    },
+    "properties": {
+        "sources": [
+            "http://www.inpe.inpe.br"
+        ]
+    }
+}
```

Comment:
I also noted the collection_name and band "common_name" (in all bands) issues on goes16-l2-cmi.json (https://github.com/brazil-data-cube/metadata/blob/master/goes16-l2-cmi.json), they should be updated, right?